class UnexpectedChallengeNameError(RuntimeError):
    pass


class UnexpectedChallengeAmountError(RuntimeError):
    pass
